datarobot_batch_scoring/detect.py

```diff
@@ -0,0 +1,91 @@
+import math
+import re
+
+
+class Detector(object):
```

Comment:
small doc string about what this class supposed to do would be awesome.
It is not very clear does it detect delimiters or encoding, for example